articles/tutorials/toc.yml

```diff
@@ -10,3 +10,5 @@
     href: intro-to-katas.md
   - name: Q# code samples
     href: https://docs.microsoft.com/samples/browse/?languages=qsharp
+  - name: MS Learn - Quantum computing fundamentals
```

Comment:
If I were unfamiliar with Learn (which I'd imagine a decent portion of people will be), I don't know if I'd ever explore this listing in the TOC. Also, while it fits for the title of the learning path, I think in the context of our documentation people might interpret "Quantum computing fundamentals" as implying something which is relatively independent from Q# or the QDK (like the "Concepts" section). 

At the very least, I think including "Learning path" in it might better convey to people what the link leads to. To also include the explicit reference to Learn, maybe something like "Learning path: MS Learn modules"?

Reply:
Good point, thanks.  I poked around other Azure features and most link to their MS Learn content from their main Hub page (which we don't have and most likely won't). However, there is some consistency among Azure features without Hub pages (Azure SQL, Web Apps, and VM) to link from TOC -> Resources -> "Build your skills with MS Learn".  Keeping this consistent wording may work for us.